.github/workflows/setup-go-cache-warmup.yml

```
@@ -0,0 +1,37 @@
+  cache-warmup:
+    name: Cache warmup (${{ matrix.target }})
+    timeout-minutes: 15
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: windows-latest
+            target: windows
+          - os: macos-latest
+            target: macos
+    runs-on: ${{ fromJson(needs.runners-config.outputs.runners-config)[matrix.os] }}
```

actionlint (1.7.8)

property "runners-config" is not defined in object type {}

(expression)